appveyor.yml

```diff
@@ -12,49 +12,41 @@ branches:
 build: off
 
 install:
-  - SET PATH=C:\Ruby%ruby_folder_version%\bin;%PATH%
+  - SET PATH=C:\Ruby%RUBY_FOLDER_VER%\bin;%PATH%
   - bundle install --retry 5
 
 environment:
   BUNDLE_WITHOUT: "benchmark:site:development"
   matrix:
-    - ruby_folder_version: "23"
-      ruby_gems_folder: "2.3.0"
+    - RUBY_FOLDER_VER: "23"
+      GEMS_FOLDER_VER: "2.3.0"
```

Comment:
In caching.

On Aug 3, 2016 02:27, "Parker Moore" <notifications@github.com> wrote:

> In appveyor.yml
> <https://github.com/jekyll/jekyll/pull/5180#discussion_r73256991>:
>
> >    - bundle install --retry 5
> >
> >  environment:
> >    BUNDLE_WITHOUT: "benchmark:site:development"
> >    matrix:
> > -    - ruby_folder_version: "23"
> > -      ruby_gems_folder: "2.3.0"
> > +    - RUBY_FOLDER_VER: "23"
> > +      GEMS_FOLDER_VER: "2.3.0"
>
> Where is the GEMS_FOLDER_VER used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5180/files/801d12b9166e11892d20242a364ec1dd6f44d1dd#r73256991>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVteNH0nAxKdi-Z_t_0kVMSf4RgqnBks5qb9JRgaJpZM4JaRox>
> .
>